packages/samples/headless-ssr-commerce/src/middleware.ts

```diff
@@ -0,0 +1,10 @@
+import {NextRequest, NextResponse} from 'next/server';
+
+export default function middleware(request: NextRequest) {
+  const response = NextResponse.next();
+  const requestHeaders = new Headers(request.headers);
+  const uuid = crypto.randomUUID();
```

Comment:
This will generate a random clientId every time, no? I think we should at least add a comment and or TODO to explain what really should happen.